pom.xml

```diff
@@ -33,6 +33,12 @@
 
```

Comment:
In the future and to be easier on my poor brain, it'd be nice to have different pull requests for task 8 and task 9! Just to keep things shorter and not have as much scrolling to do

Reply:
apologies!